tf/apis.tf

```diff
@@ -0,0 +1,6 @@
+# Cloud Run
+resource "google_project_service" "cloud_run_api" {
+  service = "run.googleapis.com"
+
+  disable_on_destroy = false
```

Comment:
To reduce risk, would it make sense to set this setting "universally" across terraform (if possible), until we know things are stable?

Reply:
That particular flag is specific to this resource. I don't see how to set a default for it. There's generally a `lifecycle { prevent_destroy=true } ` flag that you can use anywhere, but from reading about it it doesn't seem to work?

My sense is the best approach is to always do this manually, and always fully review the changes.